app/server.go

```diff
@@ -848,6 +851,11 @@ func (s *Server) Start() error {
 	if err := s.products["channels"].Start(); err != nil {
 		return errors.Wrap(err, "Unable to start channels")
 	}
+
+	if err := s.InitializePluginService(); err != nil {
```

Comment:
I think this should happen after products are started.

Reply:
Agreed, it can done once we decouple the product hooks from plugin environment.